flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/DispatcherCleanupITCase.java

```diff
@@ -280,6 +343,21 @@ private JobGraphStore createAndStartJobGraphStoreWithCleanupFailures(
             AtomicInteger actualCleanupCallCount,
             OneShotLatch successfulCleanupLatch)
             throws Exception {
+        return createAndStartJobGraphStoreWithCleanupFailures(
+                numberOfCleanupFailures,
+                throwable,
+                actualCleanupCallCount,
+                successfulCleanupLatch,
+                FutureUtils.completedVoidFuture());
+    }
+
+    private JobGraphStore createAndStartJobGraphStoreWithCleanupFailures(
```

Comment:
This seems to be only used by the method above 🤔 Please double check that this change is needed.